nmostesting/mocks/Node.py

```diff
@@ -56,6 +56,7 @@ def get_sender(self, stream_type="video"):
 
 @NODE_API.route('/<stream_type>.sdp', methods=["GET"])
 def node_video_sdp(stream_type):
+    # TODO: Should we check for an auth token here? May depend on the URL?
```

Comment:
perhaps an issue against is-10 or BCP so we dont forget?

Reply:
I've added https://github.com/AMWA-TV/nmos-authorization-practice/issues/5. Comments welcome there. It's a bit of an awkward one...